applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -67,19 +66,4 @@
             @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
         </div>
     }
-
-    @if(!Webpack.isSwitchedOn) {
-        <script>
```

Comment:
I have a question. Briefly spoke to @SiAdcock about it but it seems that [this was here to improve loading speed of interactives](https://github.com/guardian/frontend/pull/14787), but I'm not sure how much of an advantage it had over being in `jsNonBlocking` as that doesn't wait until app.js has parsed before kicking off anyway?